packages/react-components/review/beta/react-components.api.md

```diff
@@ -1082,6 +1089,7 @@ export type MessageProps = {
     showDate?: boolean;
     disableEditing?: boolean;
     onUpdateMessage?: UpdateMessageCallback;
+    onCancelMessageEdit?: CancelEditCallback;
```

Comment:
Is cancel a user triggered action or it is triggered by failed errors?

Reply:
It is a user triggered action; they need to fire it with the cancel X button on the edit box message.